backend/src/main/kotlin/org/dontstw/eventfactory/domain/Member.kt

```diff
@@ -0,0 +1,7 @@
+package org.dontstw.eventfactory.domain
+
+data class Member(
+        val uId: String,
+        var uPw: String,
+        var uName: String
```

Comment:
Member... evar....
and... variable name... evar tuna...

Reply:
Ok I will fix it ..